index.ts

```diff
@@ -1,30 +1,62 @@
-// @flow
-
 import GeocoderArcGIS from 'geocoder-arcgis'
-import lonlat from '@conveyal/lonlat'
+import type { GeoJSON } from 'geojson'
+import { toCoordinates, toString, toPoint } from '@conveyal/lonlat'
+import type { LonLatInput } from '@conveyal/lonlat'
 
-const arcGisGeocoders = {}
+type Point = {
+  x: number
+  y: number
+}
+
+type GeocodersList = {
+  [key: string]: GeocoderArcGIS
+}
 
 type Boundary = {
   rect: {
-    minLat: number,
-    minLon: number,
-    maxLat: number,
+    minLat: number
+    minLon: number
+    maxLat: number
     maxLon: number
   }
 }
 
+type Candidate = {
+  location: LonLatInput
+  attributes: Record<string, string | number>
+}
+
+type Options = {
+  searchExtent?: string
+  location?: string
+  forStorage?: boolean
+  outFields?: string
+  magicKey?: string
+  size?: number
+}
```

Comment:
I would omit this type entirely because it might produce a type definition for a public function where some or all of these are not used by a given function.